pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/exchange/HashExchangeTest.java

```diff
@@ -105,5 +106,10 @@ public Object[] getKey(Object[] input) {
     public int computeHash(Object[] input) {
       return _hashes.next();
     }
+
+    @Override
+    public String hashAlgorithm() {
+      return HASH_ALGORITHM;
```

Comment:
better represent this as enum

Reply:
Representing as enum would be hard since each implementaiton of `KeySelector` will have their own names to identify themselves which may be across different packages.

Reply:
I see, so it'll be pluggable